.gitignore

```diff
@@ -0,0 +1,4 @@
+**/*.rs.bk
+/*.data
+/*.gz
+/target/
```

Comment:
add .idea so all the intellij files are not included?

Reply:
Not all IntelliJ IDEA files are usefulness, some contains useful information but I can remove all `.idea` folder if you prefer :-)